### PR TITLE
Updated python-quiz.ts

### PR DESCRIPTION
**Pull request by anish853**
Added question on python kindly review it and merge it.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by anish853** [comment]
Kindly merge it.

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+  {
+    Question: 
+      "One line expression function in python is called as?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:4]:
```suggestion
      "In Python, what are anonymous functions called?",
```

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+    Answer: "Lambda function",
+    Distractor1: "Lambda function",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:6]: distractor are only for wrong answers

```suggestion
    Distractor1: "type function",
```

File: src/data/python-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,15 @@
...
+    Distractor2: "beta functions",
+    Distractor3: "Nothing",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:8]:
```suggestion
    Distractor3: "sorted functions",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by anish853** [comment]
Problem has been resolved. Kindly merge it.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**